app/controllers/application_controller.rb

```diff
@@ -15,4 +15,8 @@ def authenticate_user_from_token!
       sign_in user, store: false
     end
   end
+
+  def render_404
+    raise ActionController::RoutingError.new('Not Found')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.